docker_images.json

```diff
@@ -15,18 +15,18 @@
             }
         }
     },
-    "docker.io/library/python:3.11.10-slim-bullseye": {
-        "digest": "sha256:21e2cc29feabc9fa1335feb5efd98ebaf323a5963b8670a2212289b1db74cf73",
-        "mirror_digest": "sha256:ccb0c6383303002a6d15d2f52a24eb240f46a16f38984b04012a46c6a3b81560",
+    "docker.io/library/python:3.12.6-slim-bullseye": {
```

Comment:
Looks like you forgot to address https://github.com/DataBiosphere/azul/issues/5736#issuecomment-2354398118 which is a pretty significant unforced error, since I specifically remember triaging that comment with you to make sure you're aware of it.